tinygrad/runtime/ops_disk.py

```diff
@@ -71,21 +71,22 @@ class DiskDevice(Compiled):
   def __init__(self, device:str):
     if not DiskDevice._tried_io_uring_init: self._iouring_setup()
 
-    self.size: Optional[int] = None
-    self.count = 0
+    self.size = self.fd = self.count = 0
```

Comment:
Why did you remove a type? Don't save lines like this!!

Reply:
I removed the type because I don’t think annotating the type for an int assignment is necessary. The line above, ‘self.count = 0’ does not have a type annotation either. 

For more context, I changed the type of self.size from Optional[int] to an int where size = 0 instead of size = None represents that the file has not been memory mapped yet. This is because extra asserts/casts/ifs are required to work around mypy and there is a lack of helper functions like map or apply from Haskell. So I used an int and cut down on the code size. If you aren’t ok with this, let me know and I will use Nones instead of 0s in future PRs.